staging/src/k8s.io/kubectl/pkg/util/openapi/validation/schema.go

```diff
@@ -26,7 +26,6 @@ import (
 	utilerrors "k8s.io/apimachinery/pkg/util/errors"
 	"k8s.io/cli-runtime/pkg/resource"
 	"k8s.io/klog/v2"
-	schemavalidation "k8s.io/kubectl/pkg/util/openapi/validation"
 )
 
 // Schema is an interface that knows how to validate an API object serialized to a byte array.
```

Comment:
the Schema interface and the NullSchema / NoDoubleKeySchema / ConjunctiveSchema implementations aren't necessarily-related... not sure moving them under the openapi package makes sense

Reply:
The validation code should be together, since they call each other and all the structs implement the `Schema` interface. But I think you're right; it doesn't belong under `openapi/validation`, since it's not all related to OpenAPI. I've combined the two validation directories under the more generic `pkg/validation`. Please have a look.

Reply:
The cleanup of the testdata looks good... I'm a little unsure of the motivation for the move of the rest. Will it disrupt use like https://grep.app/search?q=.NewSchemaValidation%28? If so, is there a benefit to this change that makes that disruption worthwhile?

Reply:
Trying to simplify code for our move from Open API V2 to Open API V3 in kubectl. I'll add it to the description.